.github/workflows/appmap-analysis.yml

```diff
@@ -0,0 +1,36 @@
+name: appmap-analysis
+
+on:
+  pull_request:
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
```

Comment:
Looks like you need to specify the `ref` for the commit to work - https://github.com/EndBug/add-and-commit#working-with-prs

Reply:
yea - just added in the latest commit.